apigw-http-api-lambda-terraform/example-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title": "Amazon API Gateway HTTP API to AWS Lambda",
+  "description": "Create a Step Functions workflow to query Amazon Athena.",
```

Comment:
@c-los  please fix to be the correct description

Reply:
@mavi888 I fixes the description. Thank You!